### PR TITLE
Resolved #326

### LINKED ISSUE #326: feat: implement loop functionality for sponsors carousel
add loop functionality to sponsors carousel, adding autoplay would be nice too!
we are using `swiper`, you can find more information in the [swiper docs](https://swiperjs.com/)

### PR DESCRIPTION
**Pull request by atridadl**
## What issue is this referencing?

This PR resolved #326 
## Do these code changes work locally and have you tested that they fix the issue yourself?

-   [x] yes!

## Does the following command run without warnings or errors?

-   [x] yes! `npm run pr-checks`

## Have you taken a look at our [contributing guidelines](https://github.com/devedmonton/DES-Website/blob/main/.github/CONTRIBUTING.md)?

-   [x] yes!

## My node version matches the one suggested when running `nvm use`?

-   [x] yes!

### CODE REVIEW

File: components/app/SponsorCarousel.client.vue [hunk 1]
```diff
@@ -15,6 +18,11 @@ import 'swiper/css'
     </h2>
     <Swiper
       :space-between="50"
+      :autoplay="{
+        delay: 5000,
```
Comment (MandyMeindersma) [inline components/app/SponsorCarousel.client.vue:22]: nit pick, I think I maybe like 3 seconds more? Cause 5 feels like too long and then I scroll by, maaayyybe even 2.5 but that seems a little too much

Comment (atridadl) [reply components/app/SponsorCarousel.client.vue:22]: Sorry just to clarify do you mean increasing it by 3s (so 8000ms) or reducing to 3s (3000).

Comment (atridadl) [reply components/app/SponsorCarousel.client.vue:22]: Nevermind I re-read and realized what you meant... lol

**Comment by arashsheyda** [comment]
@atridadl great work! also could add `:loop="true"` so it's looped. thanks

**Comment by atridadl** [comment]
> @atridadl great work! also could you add `:loop="true"` so it's looped. thanks

Absolutely. Adding it now. Noticed that (at least locally) it looped without that. That being said, this can't hurt either.

**Comment by arashsheyda** [comment]
thanks! the `loop` property adds a loop functionality when you try to swipe manually

**Review by MandyMeindersma — APPROVED** [review]

**Comment by MandyMeindersma** [comment]
Probably the fastest code review I have ever seen in my life hahahah great job

**Comment by MandyMeindersma** [comment]
@allcontributors add @atridadl for code

**Pull request merged**